main.go

```diff
@@ -53,7 +53,9 @@ func main() {
 		log.WithField("err", err).Fatalf("Could not add html extension to the temporary file.")
 	}
 
-	open.Run("file://" + newName)
+	if err = open.Run("file://" + newName); err != nil {
+		log.WithField("err", err).Fatalf("Could not open the default viewer")
```

Comment:
@dbalan thanks for the PR! Good idea.
Can you please add some actionable content in the Fatal text and end in a dot (.)? For example: please configure your xdg-open. Cheers.